### PR TITLE
EZP-31038: add configuration for UDW for bulk add location

### PR DESCRIPTION
**Pull request by ITernovtsiy**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-31038](https://jira.ez.no/browse/EZP-31038)
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Part of https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/219
Co-Authored-By: @dew326 

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/themes/admin/content/location_view.html.twig [hunk 1]
```diff
@@ -80,12 +80,14 @@
...
                                 <h2 class="ez-fieldgroup__name">{{ 'content.view.subitems'|trans|desc('Sub-items') }}</h2>
+                                {% set udwConfigSingle = ez_udw_config('single_container', {}) %}
                                 <div class="ez-sil"
                                     data-location="{{ location.id }}"
                                     data-mfu-create-permissions-config="{{ subitems_module.content_create_permissions_for_mfu|json_encode() }}"
                                     data-items="{{ subitems_module.items }}"
                                     data-content-types="{{ subitems_module.content_type_info_list }}"
-                                    data-udw-config-bulk-move-items="{{ ez_udw_config('single_container', {}) }}"
+                                    data-udw-config-bulk-move-items="{{ udwConfigSingle }}"
+                                    data-udw-config-bulk-add-location="{{ udwConfigSingle }}"
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/content/location_view.html.twig]: You have to pass it to the subitems like the `data-udw-config-bulk-move-items` is passed in https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/public/js/scripts/admin.location.view.js#L167

Comment (ITernovtsiy) [reply src/bundle/Resources/views/themes/admin/content/location_view.html.twig]: Done, but now I'm not 100% sure I understand why we need both `data-udw-config-bulk-move-items` and `data-udw-config-bulk-add-location`  attributes if they are always the same

Comment (dew326) [reply src/bundle/Resources/views/themes/admin/content/location_view.html.twig]: `Add` and `move` are different actions and deserves a different config. The JS/React must be ready to take separate configs. If the end developer will need to have separate configs he can easily override this in the template (by changing the config key or adding context and write a subscriber). 

If we would use one config for both and wanted to split it when needed the flow probably would be probably like this:
end developer needs separate config -> contact our support -> support create a ticket -> eZ developer change the behavior (with the BC so with some fallback) -> QA -> patch for the client
if we do it now we have flow:
end developer needs separate config -> contact our support-> support gives the answer. And we do not end up with some fallbacks and technical debt in order to have BC.

So this is done for less maintenance and to give more flexibility for developers.

Comment (ITernovtsiy) [reply src/bundle/Resources/views/themes/admin/content/location_view.html.twig]: ok, thanks for the clarification

**Comment by ITernovtsiy** [comment]
all requested changes completed
failed tests are not related to this PR

**Comment by lserwatka** [comment]
@dew326 could you make a final review so we can send it to QA?

**Comment by dew326** [comment]
@lserwatka this PR looks ok, but we have to wait for the related PR (https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/219). There is an open question where this should go 2.5 or 3.0.

**Comment by lserwatka** [comment]
Master is fine. So I would leave as it is.

**Comment by katarzynazawada** [comment]
Error when clearing cache:
`Unrecognized option "limitation_value_templates" under "ezpublish.system.default"` 
I suppose, rebase is needed.

**Comment by dew326** [comment]
@katarzynazawada done

**Comment by ITernovtsiy** [comment]
added "create-location" icon requested in https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/219

**Pull request merged**